packages/jsii-pacmak/test/expected.jsii-calc/python/src/jsii_calc/submodule/nested_submodule/__init__.py

```diff
@@ -8,15 +8,20 @@
 import jsii.compat
 import publication
 
-import jsii_calc.submodule.child
+from ..child import (
+    Goodness as _Goodness_2df26737,
+)
+from .deeply_nested import (
+    INamespaced as _INamespaced_e2f386ad,
+)
 import scope.jsii_calc_base
 import scope.jsii_calc_base_of_base
 import scope.jsii_calc_lib
 
 __jsii_assembly__ = jsii.JSIIAssembly.load("jsii-calc", "0.0.0", "jsii_calc", "jsii-calc@0.0.0.jsii.tgz")
```

Comment:
Are you planning to also fix #1501 as part of this commit (i.e. always import the root module so you can use `__name__` instead of an explicit name in the `.load()` call?

Reply:
I've actually fixed it separately because this would land too late.